### PR TITLE
Redesign database => update model, update admin login

### PR DESCRIPTION
**Pull request by truongnv0211**

### CODE REVIEW

File: app/models/lesson.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class Lesson < ActiveRecord::Base
-  has_many :words, through: :lesson_words
-  has_many :lesson_words
+  has_many :words, through: :answers
```
Comment (khanhkhanh) [inline app/models/lesson.rb]: Lesson has words through answers???????????????????????????

Comment (phamkykhoi) [inline app/models/lesson.rb]: Missing  relation

File: app/models/word.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Word < ActiveRecord::Base
   belongs_to :category
-  has_many :lessons, through: :lesson_words
-  has_many :lesson_words, dependent: :destroy
+  has_many :lessons, through: :answers
```
Comment (phamkykhoi) [inline app/models/word.rb]: Missing relation

File: app/models/answer.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Answer < ActiveRecord::Base
+  belongs_to :lesson
```
Comment (phamkykhoi) [inline app/models/answer.rb]: Haven't this relation

File: app/models/lesson.rb [hunk 4]
```diff
@@ -1,5 +1,8 @@
 class Lesson < ActiveRecord::Base
-  has_many :words, through: :lesson_words
-  has_many :lesson_words
+  has_many :words
+  has_many :answers
```
Comment (phamkykhoi) [inline app/models/lesson.rb]: Haven't this relation

**Pull request closed without merging**